Subprocess.podspec

```diff
@@ -1,17 +1,17 @@
 Pod::Spec.new do |s|
   s.name         = 'Subprocess'
-  s.version      = '2.0.0'
+  s.version      = '3.0.0'
   s.summary      = 'Wrapper for NSTask used for running processes and shell commands on macOS.'
   s.license      = { :type => 'MIT', :text => "" }
   s.description  = <<-DESC
                     Everything related to creating processes and running shell commands on macOS.
                    DESC
   s.homepage     = 'https://github.com/jamf/Subprocess'
-  s.authors      = { 'Cyrus Ingraham' => 'cyrus.ingraham@jamf.com' }
+  s.authors      = { 'Cyrus Ingraham' => 'cyrus.ingraham@jamf.com', 'Michael Link' => 'michael.link@jamf.com' }
```

Comment:
Good question. If the email goes to a black hole then it's not very useful to have there. I wonder if there's a apple-natives email available for these sorts of things?

Reply:
Cyrus has a different email address now.  He used to have write access as a contributor to this open source repo, but it looks like that was removed (probably during the great permissions change to teams instead of individuals).